dt/uni04delta/edpm-pre-ceph/nodeset/kustomization.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../../../lib/dataplane
+
+patches:
+  - target:
+      kind: OpenStackDataPlaneDeployment
+      name: .*
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: edpm-deployment-pre-ceph
+
+  - target:
+      kind: OpenStackDataPlaneDeployment
+      name: .*
+    patch: |-
+      - op: replace
+        path: /spec/nodeSets
+        value:
+          - ceph-nodes
```

Comment:
Since this `kustomization.yaml` is only dealing with `OpenStackDataPlaneNodeSet`, these are not needed

Reply:
Done